tests/unit/test-class-wc-subscriptions.php

```diff
@@ -112,7 +112,7 @@ public function test_can_be_updated_to_active() {
 		$this->assertEquals( $expected, $can_be_updated, '[FAILED]: pending-cancel to wc-active.' );
 
 		// End date in the past
-		$subcription->update_dates( [ 'end' => gmdate( 'Y-m-d H:i:s', time() - DAY_IN_SECONDS ) ] );
```

Comment:
This and the 2 updates above are the only 3 uses of it. It's also in unit tests so not a concern. 👍

Reply:
Thank you. I've also checked all variable uses.